adapters/repos/db/lsmkv/memtable_flush.go

```diff
@@ -26,6 +26,30 @@ import (
 	"github.com/weaviate/weaviate/entities/diskio"
 )
 
+func (m *Memtable) flushWAL() (rerr error) {
```

Comment:
nit: if we are not using the named `rerr` error we can just remove it from method declaration